### PR TITLE
Fix viscosity calculation in Water97FluidProperties

### LINKED ISSUE #18877: Small error in water viscosity calculation
## Bug Description
There is an error in the implementation of water viscosity in `Water97FluidProperties`, where someone (me) didn't implement part of the viscosity calculation properly.

## Steps to Reproduce
Any water viscosity calculation can be off by a small amount.

## Impact
The difference is relatively minor (correcting it didn't cause any diffs in test suite).

### PR DESCRIPTION
**Pull request by cpgr**
Fixes #18877

### CODE REVIEW

**Comment by cpgr** [comment]
Pesky finite difference derivative tests... I loosened the tolerance slightly so that they hopefully pass. The numbers are virtually identical, but very small.

**Review by andrsd — DISMISSED** [review]

File: modules/fluid_properties/include/userobjects/Water97FluidProperties.h [hunk 1]
```diff
@@ -1316,13 +1316,14 @@ class Water97FluidProperties : public SinglePhaseFluidProperties
...
   /// Ordinary Water Substance
-  const std::array<int, 21> _mu_I{{0, 1, 2, 3, 0, 1, 2, 3, 5, 0, 1, 2, 3, 4, 0, 1, 0, 3, 4, 3, 5}};
-  const std::array<int, 21> _mu_J{{0, 0, 0, 0, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 3, 3, 4, 4, 5, 6, 6}};
   const std::array<Real, 4> _mu_H0{{1.67752, 2.20462, 0.6366564, -0.241605}};
-  const std::array<Real, 21> _mu_H1{
-      {5.20094e-1, 8.50895e-2, -1.08374,    -2.89555e-1, 2.22531e-1,  9.99115e-1,  1.88797,
-       1.26613,    1.20573e-1, -2.81378e-1, -9.06851e-1, -7.72479e-1, -4.89837e-1, -2.57040e-1,
-       1.61913e-1, 2.57399e-1, -3.25372e-2, 6.98452e-2,  8.72102e-3,  -4.35673e-3, -5.93264e-4}};
+  const std::array<std::array<Real, 7>, 6> _mu_Hij{
+      {{5.20094e-1, 2.22531e-1, -2.81378e-1, 1.61913e-1, -3.25372e-2, 0.0, 0.0},
```
Comment (andrsd) [inline modules/fluid_properties/include/userobjects/Water97FluidProperties.h:1321]: GCC does not like this and suggests:
```
Water97FluidProperties.h:1321:9: error: suggest braces around initialization of subobject [-Werror,-Wmissing-braces]
      {{5.20094e-1, 2.22531e-1, -2.81378e-1, 1.61913e-1, -3.25372e-2, 0.0, 0.0},
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        {                                                                     }
```

*sigh*

**Comment by andrsd** [comment]
The fail in pronghorn is unrelated and will be fixed today...

File: modules/fluid_properties/include/userobjects/Water97FluidProperties.h (hunk 1 above)
Comment (cpgr) [reply modules/fluid_properties/include/userobjects/Water97FluidProperties.h]: I've added some extra braces to hopefully satisfy the compilers.

**Review by andrsd — APPROVED** [review]
This is fine and can go in.

All the fails in BISON, pronghorn and griffin are unrelated to these changes.

File: modules/fluid_properties/include/userobjects/Water97FluidProperties.h (hunk 1 above)
Comment (andrsd) [reply modules/fluid_properties/include/userobjects/Water97FluidProperties.h]: That did it :+1:

**Pull request merged**